test/lean-riscv/Main.lean

```diff
@@ -0,0 +1,20 @@
+import LeanRiscv
+
+def main (args : List String) : IO UInt32 := do
```

Comment:
The other error you're getting is because the Sail output already includes a `main` function, so better call this differently or put it in a namespace.

Reply:
The issue is that the risc-v model has its own main function that we don't want to use here. I changed my executable model branch to not compile the `riscv_main.sail` file, so this problem should be fixed if you pull on that branch @jn80842